cloudformation/ci_resources.yml

```diff
@@ -115,6 +115,11 @@ Resources:
               - !ImportValue account-resources:PsuCAKeySecret
               - !ImportValue account-resources:PsuClientCertSecret
               - !ImportValue account-resources:PsuClientSandboxCertSecret
+          - Effect: Allow
+            Action:
+              - kms:Decrypt
+            Resource:
```

Comment:
This might not be needed in this case but should we add a `kms:Encrypt` action?

Reply:
CloudFormation doesn't set any secrets, so it doesn't need encryption.